### DISCUSSION TITLE
Find a 5-digit number

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#876] [Galaxy Trivia] [answered]
*If you place a 9 at the beginning to make a new number is four times larger than if you place the 9 at the end instead.*

**Comment by peterbaccaro** [comment]
Let's represent the original number as "x".

If we place a 9 at the beginning, the new number would be 9x.
If we place a 9 at the end, the new number would be 10x + 9.

According to the problem, if we place the 9 at the beginning, the new number is four times larger than if we place the 9 at the end. Mathematically, we can express this as:

9x = 4(10x + 9)

Expanding the right side:

9x = 40x + 36

Subtracting 40x from both sides:

9x - 40x = 36

Simplifying:

-31x = 36

Dividing both sides by -31:

x = 36 / -31

Calculating the value of x:

x ≈ -1.1613

Therefore, the original number is approximately **-1.1613**.

**Reply by ParanoidUser** [reply]
> If we place a 9 at the beginning, the new number would be 9x.

That doesn't sound quite right. 🤔

**Reply by peterbaccaro** [reply, marked as answer]
The answer is **23076**.

```
for i in range(24000):
    x = int("9" + str(i))
    y = int(str(i) + "9")
    y4 = y * 4
    if x == y4:
        print(f"Result: {i}")
```

```
Result: 23076
```

**Reply by ParanoidUser** [reply]
Ain't nothin' like an ol' good brute-force search 🤠
The mathematical reasoning still stands: `900000 + x = 4 * (10x + 9)`
